2-projects/1-todo-list/script.js

```diff
@@ -1,25 +1,101 @@
-function populateTodoList(todos) {
-  let list = document.getElementById("todo-list");
-  // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+// function populateTodoList(todos) {
+// let list = document.getElementById("todo-list");
+// Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+
+function toggleIsComplete(todo) {
+  todo.completed = !todo.completed;
+}
+
+function greenRawTodo(li, isComplete) {
+  if (isComplete) {
+    li.style.textDecoration = "line-through";
+    li.style.color = "red";
+  } else {
+    li.style.textDecoration = null;
+    li.style.color = "black";
+  }
 }
 
+function deleteTodo(li) {
+  document.querySelector("ul").removeChild(li);
```

Comment:
this will grab the first `ul` element it encounters, which may or may not be the `todo-list` you're wanting; you could make this more specific using the `id` of the todo-list `ul` (id="todo-list")